src/devices/openxrheadset/OpenXrHeadset.cpp

```diff
@@ -778,13 +654,22 @@ bool yarp::dev::OpenXrHeadset::getTouch(unsigned int /*touch_id*/, yarp::sig::Ve
     return false;
 }
 
-std::string yarp::dev::OpenXrHeadset::getInteractionProfile()
+std::string yarp::dev::OpenXrHeadset::getLeftHandInteractionProfile()
+{
+    yCTrace(OPENXRHEADSET);
```

Comment:
It this doing something?

Reply:
To be honest, this was something I "cargo-culted" from ``yarp-device-ovrheadset``. See for example https://github.com/robotology/yarp-device-ovrheadset/blob/b310d7c8588ed0c74d2d167ea41c7d804d10aa18/src/devices/ovrheadset/OVRHeadset.cpp#L271

I was thinking that it does some tracing, but I am not so sure

Reply:
I will remove them all

Reply:
I am not sure, but I think `yCTrace(..)` is basically a stream object, so those `yCTrace(..)` in the constructor originally contained something like `yCTrace(..) << "this is a costructor"` or similar.

Reply:
Removed in https://github.com/ami-iit/yarp-device-openxrheadset/pull/18/commits/582c44c7f24c41139b8efd367993d35713cdcdc0